cognee/api/v1/add/add.py

```diff
@@ -7,6 +7,12 @@
 from cognee.infrastructure import infrastructure_config
 from cognee.infrastructure.files.storage import LocalStorage
 from cognee.modules.discovery import discover_directory_datasets
+from posthog import Posthog
+
+from cognee.utils import send_telemetry
+
+posthog = Posthog(project_api_key='phc_bbR86N876kwub62Lr3dhQ7zIeRyMMMm0fxXqxPqzLm3', host='https://eu.i.posthog.com')
```

Comment:
Can we move this inside the `send_telemetry`? Then we won't need to instantiate it every time we use it, and it will abstract away `Posthog`.

Reply:
Done